src/NovelCamera.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include "NovelCamera.h"
+
+namespace NovelRT {
+NovelCamera::NovelCamera() : _cameraUboMatrix(Lazy<GeoMatrix4<float>>(std::function<GeoMatrix4<float>()>(std::bind(&NovelCamera::generateUboMatrix,
```

Comment:
I'm not sure how to improve it, but the code format of this constructor doesn't make me happy :(

Reply:
I've been fighting that for a good 30 minutes. I have no idea either.